guide/english/jquery/jquery-click-method/index.md

```diff
@@ -5,7 +5,7 @@ title: Click Method
 # Click Method
 
 
-The jQuery Click method triggers an function when an element is clicked. The function is known as a "handler" becuase it handles the click event. Functions can
+The jQuery Click method triggers an function when an element is clicked. The function is known as a "handler" because it handles the click event. Functions can
```

Comment:
Hi @mas-designs, thank you for the PR. Your change looks good to me. However, there is another typo/grammar error in the first sentence:
```
The jQuery Click method triggers an function...
```
should be
```
The jQuery Click method triggers a function...
```

Would you mind fixing it as well?